`make lagoon-kickstart` uses old rest2tasks endpoint

Thanks for the quick response over at rocket chat and opening an issue. Hoping for a quick fix 👍  
**Issue: make lagoon-kickstart fails with errorcode 7** 

Trying to install `lagoon` on OpenShift and I've been following the install guide. I have built all the images using make build and when I try the next step of `make lagoon-quickstart` all the containers are deployed and running. 

I get this error `curl -X POST http://localhost:5555/deploy -H 'content-type: application/json' -d '{ "projectName": "lagoon", "branchName": "master" }'
curl: (7) Failed to connect to localhost port 5555: Connection refused
make: *** [lagoon-kickstart] Error 7`

Looking at docker ps, i don't see any containers running on port 5555. From our chat, you've mentioned REST service has removed. Can you please advise on a fix/work around ?

--- a/Makefile
+++ b/Makefile
@@ -716,7 +716,7 @@ push-docker-host-image: build/docker-host minishift/login-docker-registry
 
 lagoon-kickstart: $(foreach image,$(deployment-test-services-rest),build/$(image))
 	IMAGE_REPO=$(CI_BUILD_TAG) CI=false docker-compose -p $(CI_BUILD_TAG) --compatibility up -d $(deployment-test-services-rest)
-	sleep 30
+	sleep 90
 	curl -X POST -H "Content-Type: application/json" --data 'mutation { deployEnvironmentBranch(input: { project: { name: "lagoon" }, branchName: "master" } )}' http://localhost:3000/graphql
 	make logs
 

--- a/Makefile
+++ b/Makefile
@@ -717,7 +717,7 @@ push-docker-host-image: build/docker-host minishift/login-docker-registry
 lagoon-kickstart: $(foreach image,$(deployment-test-services-rest),build/$(image))
 	IMAGE_REPO=$(CI_BUILD_TAG) CI=false docker-compose -p $(CI_BUILD_TAG) --compatibility up -d $(deployment-test-services-rest)
 	sleep 30
-	curl -X POST http://localhost:5555/deploy -H 'content-type: application/json' -d '{ "projectName": "lagoon", "branchName": "master" }'
+	curl -X POST -H "Content-Type: application/json" --data 'mutation { deployEnvironmentBranch(input: { project: { name: "lagoon" }, branchName: "master" } )}' http://localhost:3000/graphql
 	make logs
 
 # Publish command to amazeeio docker hub, this should probably only be done during a master deployments

